### ISSUE TITLE
Absolution | Attempting to edit skill/support gems causes crash

### ISSUE THREAD

**Issue by TheSixStringSamurai** [PathOfBuildingCommunity/PathOfBuilding#3118]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### What is the expected behaviour/value?

![poberror](https://user-images.githubusercontent.com/25860648/128653710-47fd1db3-7819-4de7-be9e-adc6c51f1a83.png)
Touching any of the drop down slots results in immediate crash/error

Esc/enter results in further error:
![poberror2](https://user-images.githubusercontent.com/25860648/128653766-879ad293-088e-4365-8629-2e0132a32e76.png)

program stops until restarted

### What is the actual behaviour/value?

Attempting to edit Absolution support gems causes a crash on activation of the drop down.

### How to reproduce the issue

Requires Minion Instability to be allocated on tree and selected from the stats window drop down for instability DPS. Works on older 2.6.0 and if I create a new link setup for Absolution.

### Build code

```shell
eNrtPWtv4ki2nzu_woo0V7saQlwPv3LTuyLkAb150CHpnpkvIwcKcMfYxA8SejX__Z4q29gmBgonPdrR3hmpG8x5v-rUcdl9_M-XqavMWRA6vvdxHzXVfYV5A3_oeOOP-_d35wfm_j__sXfcs6PJzegkdlz-yz_2PhyLz4rL5sz9uG9p-0pkB2MWfclIkd-B1Mz2ognzvSv7mx9c-MOP-9e-x_aVB9sbOlH2beDaYXhtT9nH_a9ONJjsK3Y4YN6wnV-_ZoPAn9regAX7ytR2vL4_eGTRReDHM-C1r8wd9nzlDwG0e9W7ub0r8Ha8Im-Q_cNxf8YGUcAUB65escge2pF9eOV7YQTiH174dgS8sr8vnfEk8kDtS2bP-q49vbJn4f6hBKGO7y7OnYCduWzKvMh2X1_p-64dOOEJs6dSJPuPzGURfFt-OvGflxLilEbPtRcs6Ed2pITwx8f9FrjYHrOOE4GpbDcGOxmGSZrY1LG2BenUnsKfO-CBBmy4hEdN3fwzIXtg3dEIbOjMWTtwovaEh80Sda3Uu8JexW7kzFyHR-RSrLUYnVfEkaquA77zITJOe_0clqiG1rSwar07ip9HxFp5ICsnJy44YDcGHK079pyI7Y7X853Q9-opVMRaq1Pbnz443q4qZVit-XgHrCvbs9t-KGHpS2fESpCUrAM968uRvIXklYPkYvZYAKU3kpd2J4R-lCcLRnStyOypCIkQWgd5yl5yMLyJYAlyPcGul6uBDbyBYBESadp6Ced-BCuinDWXUMTQjQ21cHDEobveIBfBMDcRvvcCFrJgXiiimqnKYaQuLmjbJJZu6oZGiEmNTURu2ZjlqmPLbBKq000Yl4wNJhewwt_aEZOLWYm44xbjoEWLYXMT0bLkG_nvKPJXOxhuhzrzWDBe9CcOc4eSChZRSqEhxURS4SLKjoqfze2wmAkYb1YnAS-5bG3oXEEzxABjyFaWWfX9MHqB_433Fe5uaK1g6seBpA8T4KLOdK3OvckidAaw5IkG7ZYN40Gp0mzwe9YgXflz0YiK9gr65xx3HeqJC523rOpA1nV3wmhFkT14PPWHY7YTk50weAd-y0KnsGQdaBLQN9AQt-2ZTHPB80mWQQ4tzWDZ8ctzWUGR12Vi--EqG10GXJrFeeyWelfNMHQNoyYhhqVpa0Ox_-iU8SL7wWVHivqCCDMYsdbvEfxnEG3Cd6yhRN4XoK_svI1Y25qdB8z7vpCmXwKXYnDmDeOAx7o0j1WMKjawU4bysXUDCN0TIlqTapa2DnF1t2Ya6yBf7VuwapiYNIlqYV3bjFWxdXkFWb112a7Bmr2LHGLF5mU74qbdyyvgyt3Ldh6lFhMTU8ObIMsdATWMpkrXwVe2K4hSTQahqvU4PhQTJv6pO535QSQutm13EAqCXW8WR4onpkPhxH9OGOwrD77vMtuDYhDELGVeguVF43cvnj7wjXvyNyhXATl1wsHvD_FoxOdK-yB2IIZiZ-fnZ-277pezbF7DxKqrDHzXtWchG37cH9luCBh8giNqVB9ajUEkBd9xojTdZKChlRwEDgulgHm-SIksklcGko9BpADBg7bL5A12t5gxHiuhrM3SlV8GWlQDKcgkjeUUFKVCzg1sYC-kIJfttRT0mctajsubOTmrXUF4J32gHDx0ZoHzEEeS0SZKjZQcfAcnpWCxxEhmhxxg2qFLCZHuYWRg06ZJLiihR5Z22ykbMQh3ybQX5eTOfmReWlizInosci1UQqiuF2wanixgOTnn5l1ZYEoAaWVVhmxkxy6__jm2XSda8MpduHqZ3BrAcJHX5348myVkeHKHYPTLy-SXlhulFDiPTPZEMy6gmPe3xHZFfBfi8x2K4ngDNx6yrpc2kEu9XfuB895XxvwGQduPPVhxPMflqsTBAIr7XcDYEaK6SvgND943DjPFVthxTlyYD8cgXgp74foPtotzfrmuRQCUkRSLTpf72fdYth98SpTuipogrLa8Jky5ItYg0QLU5ssT3ywm5JRS9B4KmesZL-Gz3nZvt5OwQ9oKlKWRtmAHWlJ3eDPqxbD4LF6T5Gt5pWnHbNot3lLpRhBthzzkDldpbvcCLXohQVf8kVIhFAT6ixOeAT3Wh32ew4KMm9iQJW57l9hKM6wXu9C2Oxx1RzPkKfqahmxE5pjvrtgJmzB7-AatVghIJ1mC9u76JPFx7g_ikI8Y3qDYKbOjydQOHnd3V8CArB-8v3ZtqCrQL_ehaXh8i27VdGT1S7GVFP39tYSrbqFk1NBvhYK0ZineX7vsF35JL7YeQt-NeT9zx6Yz1w4-xcPxLp7JCbzP4iAQ4VOR7o4LxBqR1ir7_mtCOim-85dTwLesDhuoSReelIYS-UpO5f3z0_f4ravAjtgw2ey8JVXXE5PO2gIJJaXxF85gXJXBNQ_UyPv21ob-Ij0mU5XjuNrHArBnB4lPJDK-zGe7p_WipwWyUiXl244cvWd2dL1BwGzYMPJBDs_GZJkM35AjW0nKZsqSkJJRUjJSP6gTS4ajtTWvICKra4Kq5COTdy2BYjR256cjqjeUv2pC8qVv-hCH0XJ48p469uPp1PeWh-qSwYstRiT19V1LdGfVlzT4Xn3x_11orcnNf8T0IYkJPna58MGpVasPqt1hJsTbdhAAtZT-jv5ISCgpDeW1kBkIFOgin7-E_6S9dO6ycHID3VVQq9FN3VFNRdYTAlvJ0Us-yEZArbHvLfozZ8iCnst7wlVHpJ_EtJaPLBU7MZlggfLDzMUT01wvAqL4Qz5mxUQzcINYJkUNpJrYaiDdMLQGNY0G0XVCGzrV4DM1iY4bVNcMs4EpBRysUxM1qIU1CviqqTUIRZQ24DcNrhMD6Q1Kia4CPDFJgxiaDtSxShrA0rCAJAbCiKqa3tA0neoNrGqASiyg3NBVwwKRNBWuI8PECBhiA4hhFQTRMEUgtqYDW6RaKuU0ddqwCIBQYoHABBHSQERVrYamI0o4LRBYpxgBFY3CdWwhA0SlIH1DDHuBFnwBpXSQQ8cqcIWrltkwKGiGLAQgFJDABgQBa2xwdSgyjIahm0AEtAYLYINrTTTOwiREbSATTAn2tJDW0IXM1ILLFC7Aj4hitUEIBZuAoCARMZEB1jAIyAUWNPl1BFJgC35paEjDXHQgiLhA4EEVLEJUikFEnYDzsEqFvTiODrwBBoNIhJqm2tA1FdyqUaSBOywTbKWBdvCnroL7DQJ6aIZGgCm4A_yvQURwIcEwyLLgOjIpMKSWcAEEDSiCNfCwRjFAYo0QEEejJhcEtAVc4VYdfudyq6Aa4laBGAMlNAQSIJ0z0TGoCkHEI1LXDR4EGghGVANcieFH4USDK24a3PpgFoBHIJnJ7Ql-AO0xtnDDotwy4FPUMExuBpMCBngQ_KhDfIBJdPiIIaTAgYQYsCOHTpwVHlBAWvrsQZIxSfk5G0JzM7zmySMuFK8oDD76QZ_x1EdgcCiEA7GPD6JDfMr_FOnaHfAD-jOgDLkaHn6xbVfcsuFfm8NhmCRn8mTDstVXig0G_51LBXZU08L44cPPSP2ptF8uoCSiHuayvqfw1744WbNWh6TWffWDcOLMCsJDPqiZ8Fx2Z9nTT-0XZxpPRcub_K426U98hJ4cAFOSu1aKuAUK8HaYg_4oLder5wSs0jsQ34aZewcL76xA7ygy5DHe2TF8lamU_NaJYNXiZu1P7KH_HK5xDkFF54gjXKkHkt-x9pMyEO0_19DxRq4ziBR-1IUFgDDyAwUrIQMBh6ECfUfHiXbU20CGlgsHslErE87cHDik-DMTIxWu8LXvHbTjAPbgrTiA8BkF_lRZ-HGgJCtq6jSsco1gZ8m8cTSpCtWzEklhFZ562b32WnrK-LfNIKsix3Z7seeEE86sOeOdRO7eU2fueEwRQ8OpOKGdGRARpNPMgFpZn-VmpOhhfsxA0UQGLkdzaQ6C8c5eosBW-DnFIlJd78L6pi2924_sUZb1oTJkwJeU5E2leIZg43EVKrY3LFv_QxlhZV6gJPvWhABwA9vmbs6DeswNYHvK8qCckpyU4-Esjgqv0s0CiCMnACm_W9sbV7DD271QO10IJdpqHbKhf8-5FLbEqfZ4e1CULXQOq-d31oBS4kOB5l5IJgFV9FZi6MfnFTQlBYNAr6cvDSIT2vmSWhJYwkY5pjiHHCoTcLmCq_m1fW_OAj77LS3j63lW2rFUjE9cBzxx5rGpw2rW3hXb_WeVjrJwf8lI33FR2qG_oaqqFuoqbBeW7Yi-edX8GQnTZT9UNS0ffjbXNjSAr_2UV77Vkps_G_k2jRBsD3CmEaZFlZJTLzsFQqlTyJ58UMRxxwRAnENl_OaMgkQ8c3spMxakvc3blIFdk2kslVHXKbPFcWVEIeAtCAdlZZGE0znsaB4rl7iuFzlTZ8jVy-oF79zo2zq3VS1hx0oyLdNmPj1nJhZ3XLm4TxyXKRNfPF2u2EpyyC-PwkqnbsYCZ7cLRZIncnlrsQm7VGGF2Kd-DI12vRIGG2L9v6G3Lev539nl_Xn2_Ysshe_drJSNsHOzkibFKXuJGD-r9ybulBJq7NJmiqV01w5TIO3aXG7h9Kf35yuxq-rWcpHg6hTW5cqGIsnuBL5YKr4yewa7ZqUH67Y4VREqpHq5qOohVu2d0CwtWsvJW7GUJnngReASZ8yWxEt6pFQ4ZNsOi23GLkZ7xyFdG3oy7k0x3SnEMB8l52Wk1BUmg6DXrZ8uoCp_3WXSRXeYdK0dcSU3qk6hmifnq5YhRpG11MsoeibBSMM7Q_yhwq_3yYoJc_F1VcPL1TNtxP9Ug6-XmW_lqkQWU_w8kmgicxn6h4pcMwMI0Uz8AzOgeCG5p3B8f3uZ8JtE0Sw8Ojx8fn5uzuxo4o_YC7SlzYE_zXQ7EHcOD_i9i8MW_HfSujptnT_fT_Wzp7v7i06Lsd7F_OVyfvOETPfg199Ip_XbDArZ1zZ91IOz4H4y6mmXJyeXvdHcWIyu42-__eKedKZxS395_Dr8baZ9vQwXpoPvn6bXB8-fPvm3l_bEuoo6D3ffg9lB9Onb51863wP8rLUX3_TnUW9xy-Zx5_b2X57Zsr_Ozp8-v5zcdsZj9Pg06H61vsRPj9-8x9Mru3M360ffaeh1LzvmycGX7sVT-PTkzU-9y_Pzduvy11_c3x3y0Lt6XHyfnbpg8W8L52HR_dyj_1KpdR0Pg2Hn-vv4-WDxr999M6SP5NvtAejScf2ZO7o7wJcqvsTToK2OvgXaS-he9-ePj_d3d6Z38enCaXWtC6N7TU-_uNFNfG6O8fzBv_nVoY9fOu3e58eYDmadaDa_UYPuARvf9M6d-dNX-jFxW-al4-RlU9n9oORb3oLoGHbdCnhY3FFOH9msgENIUyXgdGQSKgNHNZNIwGFsWTL0qKkblgw9gpAEmEVVGauIW9A5HFrL1TIQlpEOWaaMtlg3sCYjHzE0GStDHyrlDUKpFJymIUsmWizDMmSCBWqzjLqGpatSVtGJTBBQHdSQgVOpKsMX2n1DSj6kSoUyteS0IJZUpvETEDK-tTRTxhmIWlJ8EUa6jB7iFrmMfBRhQyqHkCWjLzIp0aUqHzZNKTvL5TgBOMkwkIoqZGC5aNZNU6ZWgZl1Gf9SinRLyn66LhV__KyJ1MphUFOKL5WyC1I1ImUXSzflVixDN6XsDIBSfFVTqrpgQ5fLSx3JwOmYSLGFPbNMmvMzR1JeI2pFNTg-LHQ78GXGBuIcGj99xj9Ak510sfxi9kWcTfvisGclZHYwmMBGmb8_IXkjw3ffn_4KK5VFmpBAGqEYYb7m82d_O8yOruzZ8ogfh00fHDbT54YjOzp1-DE68chzduSOA_4CoaySJlQ_lZ9MIzjpHo7FWb70pBz_3GfJ4bw4ZH0x6U6mBuJy4WFjDiqemE63jbc2HxAdKffX3c_3Z3t9P3b_B_DC_w2Vr87gca_NDx9Gyif2zNy9S2fqJKOXIwXtdacz1xk4UXikqHvJ8DDbUSRvnBBHMpJnA8K9fwd8lnikNrU__kbVA039-09KwN_mxMchtsfvnoVi41J6pqCIlpFK5kN_M1X1AKkqp7NtYLpXRrWVEwBe7oo5U6xm9wdy2KEPYRQpUQA4rvLAomfGPKXFOYkYuvKHyXxUiAjB1dSSh9FxFmVrIbKX0YgTmblf5uAMmx-yJBmhKh99jkGOzEmnDFRnezeBx-949J_9YLj3JSFzpPQCpuCm3lTLl0jTLF5qxwGEXbSXvMADjLj8hZTppg-wH4G19tL8OVIuDpb_74mgvmVPAGEUwwPv_TtV7Qj9gUq3A1qDAbhhsAC3R443zgFxg_zxMzI1HlarQOVh2nJgBZxjJxDTrgLDBv6jEETJ4dQshhCEkPb3tbcnlkTIehJEPaDqBhKFsIet0QFeCdjsBbf5jGab5OK-QoXk6ZBzs8wJ8muZU-SfUXUWi5wcKr_x9yVALq8DWyb7WoDsxb6b04NuTSBtK4S-FcLYCmG-StO-62dvzknqq1JYrLL3DKXleEMpLlbgKpr9Z3umtB4WYchLWbKkVS1jFairaFgO7ZUWJagT34_CmgJV6bJFqBMGS85uKOKGsEKkRMJVIpF65pUzAn4nNFpPSL0eN72-NdEWB1fF0xYPdJg7ZdGujCoiSa_DZ0v4Xbj-nD-HtTthUj-LUB1-tF4soHqRR7e7R8awu5KtCq-aikvVHa2GPDVqINnGZrhIT9ns6L5qK9eKL60OklRKyjiilp5abbPiGn7X62USqRe-ZPcoo_XLfo2YRrWNT2QCgsoA1RD7Dc2PpCfre0GvEZX13fCGcJEtyRxXMmvkQWtY7pafDcF1Sgyubd6a5ULen3XUIRJ2QvXdQGtbS6ujjUy-1A_yLSK1prFbsbYle07YMoq5pBj2iVfD-t7IGb96iyvfZDp8esBPnSX_9s7LzA9j_lKObe-KZdn5IP7YfobGX1-Yvw1WvLnrcD3X-5ANC4-H3zL--hd3sZ33dHmE-prZwcOi7Qczvj3O3liLK3CG2XvAli8ES56w3s5tUHgYPOGXvU934xtylzIm0oELwnjKhrmWctjZI-mrPJcvsC85hfuxndm340Qni-WTFNmbqrepe3bWHXt-wPK37W63UMYxUzJRWd6hSwL9mf3syeAfH2YxfXy4-g9w_R9_9E7-
```

**Comment by zao** [comment]
This seems to be from the Unleash support added via PR #2862.
The code assumes that the skill considered has a `castTime`, but that is not the case for Minion Instability.
https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/504889f25ffafc2138139b485ef4e785ec8261d1/src/Modules/CalcOffence.lua#L579

**Comment by Wires77** [comment]
Fixed in 2.8.0
